account_loan_vendor_payment/models/account_loan.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class AccountLoan(models.Model):
+    _inherit = "account.loan"
+
+    invoice_ids = fields.Many2many(comodel_name="account.move")
```

Comment:
may be is better to reference `payment_ids` as loan is opened against the payment. What u think?

Reply:
@odooNextev gentle reminder :)

Reply:
I added payment_ids reference, but I didn't remove invoice_ids reference because is used to move from invoice to loan using a smart button.

Reply:
> I added payment_ids reference, but I didn't remove invoice_ids reference because is used to move from invoice to loan using a smart button.

you can get invoice from payments, kind of `payments.mapped('invoice_id')`; would be enough?

Reply:
If I try to remove invoice_ids from model, I had some problem in account.move, because loan_ids is empty and I couldn't get loans from invoice, because payment_id reference is always empty.

Reply:
i think you already have `move_ids` referencing `account.move` on `account.loan`, would that do the job without adding `invoice_ids`?

https://github.com/OCA/account-financial-tools/blob/fe88b783b1919c4e8cafbd436b4aefc13b3328ad/account_loan/model/account_loan.py#L225